package.bzl

```diff
@@ -51,6 +51,13 @@ def rules_typescript_dependencies():
         sha256 = "a13af63e928c34eff428d47d31bafeec4e38ee9b6940e70bf2c9cd47184c5c16",
     )
 
+    _maybe(
+        http_archive,
+        name = "go_devserver",
+        urls = ["https://github.com/bazelbuild/rules_typescript/releases/download/%s/devserver_binaries.zip" % VERSION],
```

Comment:
I don't think we should do it this way. It means the user downloads the rules_typescript as a zip from github, then we later download another zip from github.
Instead I think we want the entire rules_typescript to be distributed as a zip file that we publish, which contains the current WORKSPACE and BUILD/.bzl files

Reply:
I'll update the dependency to point to `//devserver_binaries`.